draft-ietf-dnsop-svcb-httpssvc.md

```diff
@@ -139,12 +139,11 @@ Consider a simple zone of the form
 
 The domain owner could add records like
 
-    simple.example. 7200 IN HTTPSSVC 1 . alpn=h3 ...
-                            HTTPSSVC 2 . alpn=h2 ...
+    simple.example. 7200 IN HTTPSSVC 1 . transport=quic ...
 
-The presence of these records indicates to clients that simple.example
-supports HTTPS, and the key=value pairs indicate that it prefers HTTP/3
-but also supports HTTP/2.  The records can also include other information
+The presence of this record indicates to clients that simple.example
+supports HTTPS, and the key=value pairs indicate that it supports QUIC
```

Comment:
Perhaps "supports HTTPS-over-QUIC (HTTP/3) in addition to HTTPS over TLS" ?

Reply:
Adding a "meaning" column to the registry may make sense.

It seems like we're making enough changes to Alt-Svc here that we may need to split some of these out to an alt-svc-bis draft?   Switching Alt-Svc to use ALPN=any but focusing on transport may make sense.  At a minimum we likely should include a section here on how filtering works for Alt-Svc in a world where records are like:

       Alt-Svc: any="pool.service.example:443" transport=quic, any="pool.service.example:443" transport=tls 

As part of this change we should include a reminder that clients using HTTPSSVC or AltSvc with "any" MUST still send TLS SNI (or ESNI when esniconfig is specified).

Going back to a question from @MikeBishop, do we still want a hinting/filtering mechanism for ALPN?  This can be useful for letting clients using an experimental protocol know that it is available on a pool with a different name.  That has been one Alt-Svc use-case for ALPN.  Or does this add too much complexity?